src/Microsoft.Diagnostics.Monitoring/DiagnosticsEventPipeProcessor.cs

```diff
@@ -131,7 +131,7 @@ private void HandleLoggingEvents(EventPipeEventSource source)
             var logActivities = new Dictionary<Guid, LogActivityItem>();
             var stack = new Stack<Guid>();
 
-            source.Dynamic.AddCallbackForProviderEvent(LoggingSourceConfiguration.MicrosoftExtensionsLoggingProviderName, "ActivityJsonStart/Start", (traceEvent) =>
+            source.Dynamic.AddCallbackForProviderEvent(LoggingSourceConfiguration.MicrosoftExtensionsLoggingProviderName, "ActivityJson/Start", (traceEvent) =>
```

Comment:
I believe there is a couple of hard-coded `ActivityJsonStop/Stop`s too. Could you please update those too? Thanks Adam!